### PR TITLE
Update api_walkthrough.rst

### PR DESCRIPTION
**Pull request by jakjothi**

### CODE REVIEW

File: docs/api_walkthrough.rst [hunk 1]
```diff
@@ -87,8 +87,9 @@ Successfully opening a channel will return the following information::
         "partner_address": "0x61c808d82a3ac53231750dadc13c777b59310bd9",
         "token_address": "0x9aBa529db3FF2D8409A1da4C9eB148879b046700",
         "balance": 1337,
-        "state": "open",
-        "settle_timeout": 600
+        "state": "opened",
+        "settle_timeout": 600,
+        "reveal_timeout": 10
```
Comment (LefterisJP) [inline docs/api_walkthrough.rst]: There is no need to specify reveal timeout. It's kind of an esoteric option which we would not like to advertise too much as being available so it was left out on purpose. Can you remove it?

File: docs/api_walkthrough.rst [hunk 2]
```diff
@@ -239,7 +240,8 @@ When successful this will give a response with a channel object where the state
         "token_address": "0xc9d55C7bbd80C0c2AEd865e9CA13D015096ce671",
         "balance": 350,
         "state": "closed",
-        "settle_timeout": 600
+        "settle_timeout": 600,
+        "reveal_timeout": 10
     }
 
 Notice how the ``state`` is now set to ``"closed"`` compared to the previous channel objects where it was ``"open"``.
```
Comment (LefterisJP) [inline docs/api_walkthrough.rst]: Since you change `"open"` to `"opened"` please change it here too.

File: docs/api_walkthrough.rst (hunk 1 above)
Comment (jakjothi) [reply docs/api_walkthrough.rst]: Sure

**Comment by LefterisJP** [comment]
@jakjothi thank you but can you rebase them and squash them both in 1 commit?

**Pull request merged**